docs/guides/advanced/link-to-code-source.md

```diff
@@ -0,0 +1,45 @@
+!!! warning "Experimental"
+    This feature is experimental and may change in the future.
+
+    Please give us feedback on [Slack][slack].
+
+We support linking to the source code on GitHub, or any other VCS provider.
+
+![Link to GitHub](../../images/guide/link-to-github.gif)
+
+## Usage
+
+To enable source code linking with Logfire, you need to configure your project. Here's how:
```

Comment:
```suggestion
To enable source code linking with Logfire, you need to configure your project's code source. Here's an example:
```

Reply:
```suggestion
Here's an example:
```

This configuration is not at the project level anyway.